### PR TITLE
docs(web): add new page for tabdatalist

### PR DESCRIPTION
**Pull request by SavvyShah**
Adds a page for a new component called `TabDataList`.

https://www.loom.com/share/fbcb559d1abb49369148c0be94c3c4fb

### CODE REVIEW

**Review by bietkul — CHANGES_REQUESTED** [review]
@SavvyShah Docs content is not relevant to `TabDataList`.
Incorrect description, example uses. `SingleDataList` name repeats several times.

File: content/docs/reactivesearch/v3/list/tabdatalist.md [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+      }
+    }
+  }
+/>
+```
+
+-   **className** `String`
+    CSS class to be injected on the component container.
+-   **style** `Object`
+    CSS styles to be applied to the **SingleDataList** component.
```
Comment (siddharthlatest) [inline content/docs/reactivesearch/v3/list/tabdatalist.md]:
```suggestion
    CSS styles to be applied to the **TabDataList** component.
```

File: content/docs/reactivesearch/v3/list/tabdatalist.md [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+-   **customQuery** `Function`
+    takes **value** and **props** as parameters and **returns** the data query to be applied to the component, as defined in Elasticsearch Query DSL.
+    `Note:` customQuery is called on value changes in the **SingleDataList** component as long as the component is a part of `react` dependency of at least one other component.
```
Comment (siddharthlatest) [inline content/docs/reactivesearch/v3/list/tabdatalist.md]:
```suggestion
    `Note:` customQuery is called on value changes in the **TabDataList** component as long as the component is a part of `react` dependency of at least one other component.
```

File: content/docs/reactivesearch/v3/list/tabdatalist.md [hunk 3]
```diff
@@ -0,0 +1,228 @@
+---
+title: 'TabDataList'
+meta_title: 'TabDataList'
+meta_description: 'TabDataList creates a multiple selection based list UI component that is connected to a database field.'
```
Comment (siddharthlatest) [inline content/docs/reactivesearch/v3/list/tabdatalist.md]:
```suggestion
meta_description: 'TabDataList creates a multiple selection based list UI component that is connected to a search index field.'
```

**Pull request merged**